examples/apache-maven/custom-pipelines-hello-world/decodable-resources.yaml

```diff
@@ -0,0 +1,69 @@
+---
+kind: secret
+metadata:
+  name: api-key-secret
+  description: API Key for purchase orders REST source
+spec_version: v1
+spec:
+  value_literal: "my very secret API key"
+---
+kind: connection
+metadata:
+  name: purchase-orders-source
+  description: A REST source for purchase orders
+spec_version: v2
+spec:
+  connector: rest
+  type: source
+  properties:
+    format: json
+    parse-error-policy: FAIL
+    api-key: api-key-secret
+  stream_name: purchase-orders
+  schema_v2:
+    fields:
+      - kind: physical
+        name: customer_name
+        type: string
+  execution:
+    active: true
+    task_count: 1
+    task_size: M
+---
+kind: stream
+metadata:
+  name: purchase-orders
+  description: Purchase order data
+spec_version: v1
+spec:
+  schema_v2:
+    fields:
+      - kind: physical
+        name: customer_name
+        type: string
+---
+kind: stream
+metadata:
+  name: purchase-orders-processed
+  description: Processed purchase order data
+spec_version: v1
+spec:
+  schema_v2:
+    fields:
+      - kind: physical
+        name: customer_name
+        type: string
+---
+kind: pipeline
+metadata:
+    name: customer-processor
+spec_version: v2
+spec:
+    type: JAVA
+    job_file_path: target/custom-pipelines-hello-world-0.1.jar
+    properties:
+        flink_version: 1.16-java11
+    execution:
+      active: false
```

Comment:
is it on purpose that the pipline isn't activated by default in this example? connection gets activated right away and so could the pipeline or am I missing something maybe?

Reply:
Ah yeah, good catch, will change that.